cluster_kwargs.yaml

```diff
@@ -33,6 +33,13 @@ spill_cluster:
   worker_disk_size: 64
   worker_vm_types: [m6i.large]  # 2CPU, 8GiB
 
+# For tests/benchmarks/test_xarray.py
+group_reduction_cluster:
+  n_workers: 20
+  worker_vm_types: [m6i.xlarge]  # 4CPU, 16GiB
+  spot_policy: spot_with_fallback
```

Comment:
even though we care about memory pressure, this causes the runtime to flake around unnecessarily and our regression detection would trigger unnecessarily.

likewise, if spot instances fail, data replication might also push memory pressure up

```suggestion
```

Reply:
we have tests that run much longer than 50s so I don't think we have to be cost sensitive for this specific addition